tests/Australia/ACT/QueensBirthdayTest.php

```diff
@@ -52,13 +51,12 @@ public function testHoliday($year, $expected)
         );
     }
 
-
     /**
      * Returns a list of test dates
      *
      * @return array list of test dates for the holiday defined in this test
      */
-    public function HolidayDataProvider(): array
+    public function HolidayDataProvider()
```

Comment:
Yasumi is supporting PHP7 going forward, so please use/keep any return type declarations.

Reply:
Sure, but this is probably going to be needed in a lot of the other ones I've written. I copied and pasted a lot. I'll go through and try to get them all.